Projects/Foundations/Schemas/Docs-Nodes/W/Workspace/Workspace/workspace.json

```diff
@@ -516,6 +606,11 @@
                     "language": "DE",
                     "text": "Wenn Sie einen Plugin Workspace laden, ändern und speichern, bleibt die eigentliche Plugin-Datei intakt, d. h. das Plugin wird nicht geändert. Stattdessen wird eine neue Datei im Ordner My-Workspaces erstellt. Diese Datei ist eine Kopie des Plugin-Arbeitsbereichs mit den von Ihnen vorgenommenen Änderungen. Die Kopie wird unter dem Namen der Node  Workspace gespeichert, der in der Regel derselbe ist wie der Name des Plugins. Um den Workspace unter einem anderen Namen zu speichern, müssen Sie den Namen der Workspace Node ändern und speichern.",
                     "updated": 1639345443024
+                },
+                {
+                    "language": "EL",
+                    "text": "Όταν φορτώνετε, τροποποιείτε και αποθηκεύετε έναν Plugin Workspace (Χώρο Εργασίας Πρόσθετων), το πραγματικό αρχείο του πρόσθετου παραμένει άθικτο, δηλαδή, το πρόσθετο δεν τροποποιείται. Αντίθετα, δημιουργείται ένα νέο αρχείο στο φάκελο My-Workspaces. Αυτό το αρχείο είναι ένα αντίγραφο του χώρου εργασίας του πρόσθετου που περιλαμβάνει τις τροποποιήσεις που εισαγάγατε. Το αντίγραφο αποθηκεύεται με το όνομα του κόμβου Χώρου Εργασίας, το οποίο είναι συνήθως το ίδιο όνομα με το πρόσθετο. Για να αποθηκεύσετε το χώρο εργασίας με διαφορετικό όνομα, πρέπει να αλλάξετε το όνομα του κόμβου του χώρου εργασίας και να το αποθηκεύσετε.",
+                    "updated": 1641830110805
```

Comment:
For me the "text" line looks a bit more yellow than the other ones. That shouldn`t make a difference but what is the reason or does it only look like that to me?